xray/services/scan.go

```diff
@@ -113,7 +113,7 @@ func (ss *ScanService) ScanGraph(scanParams XrayGraphScanParams) (string, error)
 		}
 		scanParams.MultiScanId = multiScanId
 		if err = os.Setenv("JF_MSI", multiScanId); err != nil {
-			errorutils.CheckErrorf("failed setting msi as environment variable. Cause: %s", err.Error())
+			log.Warn(errorutils.CheckErrorf("failed setting msi as environment variable. Cause: %s", err.Error()))
```

Comment:
We try to reduce the usage of `log.Error` and `log.Warn` to the minimum.
I suggest changing the above line to - 
```
 return errorutils.CheckErrorf("failed setting msi as environment variable. Cause: %s", err.Error())
```
You probably wouldn't like to have code flow continue after such an error.

Reply:
Suggestion: If we'll do `log.Debug` - would it be a good solution?
We wouldn't want to drop it - but do have the possibility to debug when necessary

Reply:
I changed the log to Debug, since we don't want to stop the whole scan.